Eastern-Research-Group/mywaterway
# Feature/714 climate tab wire layers

## Related Issues:
* [HMW-726](https://jira.epa.gov/browse/HMW-726)

## Main Changes:
* Finished building out the layers and statuses on the extreme weather tab.
* Split up the "Historical Risk and Potential Future Scenarios" section into two separate sections with the same label.  
  * EPA to provide us with distinct names for these sections. 
* Added disclaimer text and tooltips, discussed in last meeting. There was also an email from Cristina detailing this.
* Moved a lot of the config for this tab to an S3 config file. I might move more stuff, if I have time. 
* Added a county dropdown for viewing data based on any county that intersects with the selected HUC.
  * This dropdown defaults to the county that is displayed on the "Drinking Water" tab.
* Added combined sewer overflows layer, which wasn't in the original mockup.

## Steps To Test:
1. Navigate to http://localhost:3000/community/020802011001/extreme-weather
2. Verify that layers load
3. Verify the switches make the layers visible/hidden
4. Verify that toggling layer visibilty from the layer list also toggles the corresponding switch
5. Block one of the services
6. Verify the status text, displays `N/A`
7. Verify the popups work on various layers
    * Note: Popups do not work on the historical/future Coastal Flooding layer, because it is a TiledImageryLayer
8. Verify the drinking water tab still works

## TODO:
- [ ] Need to get input from EPA on what layers, if any, they want on by default.
- [ ] Need to get input from EPA on naming for the two "Historical Risk and Potential Future Scenarios" sections.
- [ ] Need to get input from EPA on popup content (probably after this is merged and they have time to play with it). The main layers for this are Combined Sewer Overflows, Above/Below Ground Storage Tanks, and Wells. I put in some basic popups for these layers, but EPA may want more info shown there.
- [x] Move more stuff to S3 config. 
- [ ] Research getting popups working on the historical/future Coastal Flooding layer. This layer is a TiledImageryLayer, which doesn't seem to have as good popup support.
